docs/_static/custom.css

```diff
@@ -1,15 +1,24 @@
 @import url("https://fonts.googleapis.com/css?family=Montserrat&display=swap");
-@import url("https://fonts.googleapis.com/css?family=Roboto+Mono&display=swap");
 
 body {
   font-family: "Montserrat", sans-serif;
 }
 
-.rst-content pre.literal-block, .rst-content div[class^="highlight"] pre, .rst-content .linenodiv pre {
-  font-family: "Roboto Mono", monospace;
+.md-logo {
+    width: 40px;
+    height: 40px;
+    padding-bottom: 2px;
+    padding-top: 2px;
+}
+.md-logo img {
+    width: 40px;
+    height: 40px;
 }
 
-code {
-  font-family: "Roboto Mono", monospace;
+.md-header, .md-footer-nav {
+    background-image: linear-gradient(45deg, rgb(0, 150, 225) 0%, rgb(27, 141, 226) 24%, rgb(42, 125, 227) 53%, rgb(53, 112, 227) 78%, rgb(53, 112, 227) 100%);
 }
 
+.md-header-nav__title {
+    font-size: .85rem;
+}
```

Comment:
Where do these overrides come from?

Reply:
@hiddeco and I took inspiration from https://github.com/openfaas/docs/blob/master/mkdocs.yml and https://github.com/openfaas/docs/blob/master/docs/stylesheets/extra.css and decided in https://github.com/fluxcd/flux/pull/2919#issuecomment-600177513 to use the above.